apollo/build.gradle.kts

```diff
@@ -390,20 +392,30 @@ afterEvaluate {
     }
     // Disable publish of targets
     if (os.isMacOsX) {
-        tasks.named("publishIosX64PublicationToSonatypeRepository") {
-            this.enabled = false
+        if (tasks.findByName("publishIosX64PublicationToSonatypeRepository") != null) {
+            tasks.named("publishIosX64PublicationToSonatypeRepository") {
+                this.enabled = false
+            }
```

Comment:
Wonder with this new if's you don't need the `if(os.isMacOsX)`

Reply:
We need the if condition because we are also running the project on a `Linux` environment in which these Gradle tasks do not exist.